### PR TITLE
maxquant: checks on exp. design template + documentation + bugfix for identifier parse rules

### LINKED ISSUE #392: maxquant regexp problems
```
Template of the MaxQuant parameter file '/gpfs1/work/songalax/galaxy/database/jobs_directory/032/32853/working/mqpar.xml' is created.
Traceback (most recent call last):
  File "/gpfs1/data/galaxy_server/galaxy/shed_tools/toolshed.g2.bx.psu.edu/repos/galaxyp/maxquant/8bac3cc5c5de/maxquant/mqwrapper.py", line 113, in <module>
    description=args['description_parse_rule'])
  File "/gpfs1/data/galaxy_server/galaxy/shed_tools/toolshed.g2.bx.psu.edu/repos/galaxyp/maxquant/8bac3cc5c5de/maxquant/mqparam.py", line 232, in add_fasta_files
    fastaentry = ET.fromstring(fasta)
  File "/gpfs1/data/galaxy_server/galaxy/database/dependencies/_conda/envs/mulled-v1-88194d212d404fa1b66e7626414cd5e03962c42532a28f317b7bc004d3e11dd9/lib/python3.7/xml/etree/ElementTree.py", line 1315, in XML
    parser.feed(text)
xml.etree.ElementTree.ParseError: mismatched tag: line 3, column 38
```

Problem was that the user used `<` and `>` characters in the regular expressions. I guess some validators or replacements are necessary.

### PR DESCRIPTION
**Pull request by dglaetzer**
fixes #392 and #393. For discussion on it and a demonstration of my magic git skills see #394

### CODE REVIEW

File: tools/maxquant/maxquant.xml [hunk 1]
```diff
@@ -310,17 +310,29 @@ This tool is a wrapper for MaxQuant v@VERSION@. The current version of the wrapp
...
                     File1        1          E1       False
                     File2        2          E1       False
+                    ghost       234        none
                     File3        3          E1       False
-                    ...
-                    ...
+                    File4                   E2       true
+                    File5                   E1
+
+        - This is the counter-example:
```
Comment (bgruening) [inline tools/maxquant/maxquant.xml]: counter example means this will not work?

Comment (dglaetzer) [reply tools/maxquant/maxquant.xml]: yes, one error per line

**Comment by bgruening** [comment]
Merci!

**Comment by bernt-matthias** [comment]
Cool. Thanks, installed it right away. Don't worry about git, your approach is very common: https://xkcd.com/1597/

**Pull request merged**